en/ios/users.mdown

```diff
@@ -485,10 +485,19 @@ There's also two code changes you'll need to make. First, add the following to y
 import FBSDKCoreKit
 import ParseFacebookUtilsV4
 
-// AppDelegate.swift
-func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
-  Parse.setApplicationId("parseAppId", clientKey:"parseClientKey")
-  PFFacebookUtils.initializeFacebookWithApplicationLaunchOptions(launchOptions)
+// AppDelegate.swift (Swift 3)
+func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -&gt; Bool {
```

Comment:
Is &gt; necessary here? It should render fine within the code block, and it looks like you didn't need to use this encoding later in the same file.